crossbeam-epoch/src/lib.rs

```diff
@@ -68,31 +68,33 @@ cfg_if! {
     if #[cfg(feature = "nightly")] {
         extern crate alloc;
     } else {
-        mod alloc {
-            extern crate std;
-            pub use self::std::*;
-        }
+        #[cfg(feature = "std")]
+        extern crate std as alloc;
     }
 }
 
-extern crate arrayvec;
-extern crate crossbeam_utils;
-#[macro_use]
-extern crate memoffset;
-#[macro_use]
-extern crate scopeguard;
+cfg_if! {
+    if #[cfg(any(feature = "std", feature = "nightly"))] {
```

Comment:
`alloc` crate is already stable on rust 1.36, I think it makes sense to add the `alloc` feature and replace it with `#[cfg(any(feature = "alloc", feature = "std"))]`.

Reply:
Added 22d1442.